resources/user-agents/apps/facebook/facebook-app-4-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Facebook App #MAJORVER#.#MINORVER#",
-  "versions": [29, 28, 27, 26, 25, 24, "23.1", 23, 22, 21, "20.1", 20, 19, "18.1", 18, 17, 16, 15, "14.9", "14.8", "14.7", "14.6", "14.5", "14.4", "14.3", "14.2", "14.1", 14, "13.1", 13, "12.1", 12, 11, 10, 9, 8, 7, "6.9", "6.8", "6.7", "6.6", "6.5", "6.4", "6.3", "6.2", "6.1", 6, "5.6", "5.5", "5.4", "5.3", "5.2", "5.1", 5, "4.1", 4, "3.9", "3.8", "3.7", "3.6", "3.5", "3.4", "3.3", "3.2", "3.1", 3, "2.3", "2.2", "2.1", 2],
+  "versions": [31, 30, 29, 28, 27, 26, 25, 24, "23.1", 23, 22, 21, "20.1", 20, 19, "18.1", 18, 17, 16, 15, "14.9", "14.8", "14.7", "14.6", "14.5", "14.4", "14.3", "14.2", "14.1", 14, "13.1", 13, "12.1", 12, 11, 10, 9, 8, 7, "6.9", "6.8", "6.7", "6.6", "6.5", "6.4", "6.3", "6.2", "6.1", 6, "5.6", "5.5", "5.4", "5.3", "5.2", "5.1", 5, "4.1", 4, "3.9", "3.8", "3.7", "3.6", "3.5", "3.4", "3.3", "3.2", "3.1", 3, "2.3", "2.2", "2.1", 2],
```

Comment:
Just so I'm clear, every time that Facebook releases a new major version of their app, this array has to be updated, otherwise browscap will identify it as a Webview instead of Facebook?